### PR TITLE
Add verible package

### PR DESCRIPTION
**Pull request by tgorochowik**
Signed-off-by: Tomasz Gorochowik <tgorochowik@antmicro.com>

### CODE REVIEW

File: verible/build.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+sudo add-apt-repository ppa:ubuntu-toolchain-r/test -y
+sudo apt update
+sudo apt install -y gcc-8 g++-8
+
+export CC=gcc-8
+export CXX=g++-8
+
+wget https://github.com/bazelbuild/bazel/releases/download/0.29.1/bazel-0.29.1-installer-linux-x86_64.sh
+chmod +x bazel-0.29.1-installer-linux-x86_64.sh
+./bazel-0.29.1-installer-linux-x86_64.sh --prefix=$PREFIX
```
Comment (tgorochowik) [inline verible/build.sh]: I was unable to build it locally with bazel 0.20.0 (which is available as a conda package - thus installing this by hand like that).

File: verible/build.sh [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+set -e
+set -x
+
+sudo add-apt-repository ppa:ubuntu-toolchain-r/test -y
+sudo apt update
+sudo apt install -y gcc-8 g++-8
+
+export CC=gcc-8
+export CXX=g++-8
```
Comment (tgorochowik) [inline verible/build.sh]: Installing gcc-8 to get c++17 features - this method is copied from the Slang build script.

File: verible/build.sh [hunk 3]
```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -e
+set -x
+
+sudo add-apt-repository ppa:ubuntu-toolchain-r/test -y
```
Comment (mithro) [inline verible/build.sh]: A build.sh receipe should never call `sudo`

Comment (tgorochowik) [reply verible/build.sh]: I actually copied these lines from the Slang recipe. I guess we should remove it there too then

**Comment by mithro** [comment]
@tgorochowik - I have updated this pull request to use the same method of using the local gcc compiler as how I changed the slang repo in https://github.com/SymbiFlow/conda-packages/pull/60/commits/3269832cddf7a68b687a3f31f9e652ef338e6a4a

**Pull request merged**